packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -0,0 +1,288 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { color, reduceMotion, rtl } from '../../utils/style-mixins';
+
+const rangeHeight = () => css( { height: 30, minHeight: 30 } );
+
+export const Root = styled.span`
+	-webkit-tap-highlight-color: transparent;
+	box-sizing: border-box;
+	cursor: pointer;
+	align-items: flex-start;
+	display: inline-flex;
+	justify-content: flex-start;
+	padding: 0;
+	position: relative;
+	touch-action: none;
+	width: 100%;
+
+	${rtl( {
```

Comment:
I think, I'm not sure that the `rtl` mixin is based on the direction of the document. Whicih makes me wonder whether this style here is needed since the direction is already applied above?

Reply:
Ah, you are correct! I had thought I needed it during early development of this. I just tried removing it, and it works as expected 👍